docs/api-reference/state/observer.md

```diff
@@ -73,4 +73,45 @@ numCoins:set(25) -- prints 'coins is now: 25'
 
 -- always clean up your connections!
 disconnect()
+```
+
+-----
+
+<p class="fusiondoc-api-pills">
+	<span class="fusiondoc-api-pill-since">since v0.3</span>
+</p>
+
+### :octicons-code-24: Observer:onBind()
+
+Works the same as `:onChange()`, however it run's the specified callback
+immediately.
+
+This modified method is better to be used when using the same function to initialise an
+observer and to watch changes for it.
+
+```Lua
+(callback: () -> ()) -> (() -> ())
+```
+#### Parameters
+
+- `callback` - The function to call when a change is observed **and** when the observer is created
```

Comment:
The 'Connection memory leaks' warning block from 'onChange' should be copied into here too:

![image](https://user-images.githubusercontent.com/11815374/216746392-161c6c54-84d2-4363-a9bb-79c13df1ab1b.png)